build.gradle

```diff
@@ -81,13 +85,36 @@ subprojects {
             implementation 'org.odpi.egeria:open-connector-framework:2.5-SNAPSHOT'
             implementation 'org.slf4j:slf4j-api'
         }
+
+        testImplementation "org.junit.jupiter:junit-jupiter-params:5.0.0-M4"
+        testImplementation "org.junit.jupiter:junit-jupiter-api:5.0.0-M4"
+
+        testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.0.0-M4"
+
+        test {
+            useJUnitPlatform {
+                includeEngines 'junit-jupiter', 'junit-vintage'
+            }
+            filter {
+                //include all tests from package
+                includeTestsMatching "org.gradle.internal.*"
```

Comment:
It's interesting the doc uses this as an example - I had presumed it was only to show how to pick tests from
a selected package. We probably don't need it (though it would be harmless)